packages/framework/collabspaces/src/collabSpaces.ts

```diff
@@ -590,18 +590,21 @@ export class CollabSpacesRuntime
 		const col = colArg + 1;
 		assert(row > 0 && col > 0, "arguments");
 		const cellValue = this.matrix.getCell(row, col);
+		const rowId = this.matrix.getCell(row, 0) as unknown as uuidType;
+		const colId = this.matrix.getCell(0, col) as unknown as uuidType;
```

Comment:
why did you move this? This is not performant (in case we have no value). Is this for extra validation?